apps/server/lib/google-drive/src/client.ts

```
@@ -0,0 +1,814 @@
+	private async retryRequest(
+		options: FinalRequestOptions,
+		retriesRemaining: number,
+		requestLogID: string,
+		responseHeaders?: Headers | undefined
+	): Promise<APIResponseProps> {
+		let timeoutMillis: number | undefined;
+
+		// Note the `retry-after-ms` header may not be standard, but is a good idea and we'd like proactive support for it.
+		const retryAfterMillisHeader = responseHeaders?.get("retry-after-ms");
+		if (retryAfterMillisHeader) {
+			const timeoutMs = parseFloat(retryAfterMillisHeader);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)